src/cascadia/TerminalControl/TermControl.cpp

```diff
@@ -938,10 +958,21 @@ namespace winrt::Microsoft::Terminal::TerminalControl::implementation
 
     void TermControl::_KeyHandler(Input::KeyRoutedEventArgs const& e, const bool keyDown)
     {
+        if (e.OriginalKey() == VirtualKey::Enter)

```

Comment:
Can you please document here why aren't we handling Enter in the terminal control? I guess it works, but I am not sure why :blush: